app/cdap/components/DirectiveInput/Components/AutoComplete/index.tsx

```diff
@@ -0,0 +1,236 @@
+/*
+ * Copyright © 2017 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { Box, Typography } from '@material-ui/core';
+import MyDataPrepApi from 'api/dataprep';
+import {
+  IAutoCompleteProps,
+  IOnRowClickValue,
+} from 'components/DirectiveInput/Components/AutoComplete/types';
+import { defaultFuseOptions } from 'components/DirectiveInput/constants';
+import { IUsageDirectives } from 'components/DirectiveInput/types';
+import ee from 'event-emitter';
+import Fuse from 'fuse.js';
+import reverse from 'lodash/reverse';
+import Mousetrap from 'mousetrap';
+import React, { useEffect, useState } from 'react';
+import globalEvents from 'services/global-events';
+import NamespaceStore from 'services/NamespaceStore';
+import styled from 'styled-components';
+import uuidV4 from 'uuid/v4';
+
+const DirectiveLabel = styled(Typography)`
+  font-style: normal;
+  font-weight: 600;
+  font-size: ${(props) => (props.isDescription ? '14px' : '16px')};
```

Comment:
use 2 variations of DirectiveLabel
keep all styled common in DirectiveLabel

create 2 new styled components with appropriate font sizes and use them in the JSX

Reply:
Done